checker-qual/src/main/java/org/checkerframework/checker/regex/qual/PolyRegex.java

```diff
@@ -11,9 +11,10 @@
 /**
  * A polymorphic qualifier for the Regex type system.
  *
- * <p>Any method written using {@link PolyRegex} conceptually has two versions: one in which every
- * instance of {@link PolyRegex} has been replaced by {@link Regex}, and one in which every instance
- * of {@link PolyRegex} has been replaced by {@link UnknownRegex}.
+ * <p>Any type annotated by {@link PolyRegex} conceptually can be replaced by the following
```

Comment:
What is missing here is that all `PolyRegex` in a method signature are replaced with the same qualifier. This change makes it sound like each occurrence can be replaced independently.

Reply:
Noted. I will change the words.